sympy/functions/special/delta_functions.py

```diff
@@ -40,16 +40,19 @@ class DiracDelta(Function):
 
     DiracDelta function has the following properties:
 
-    1) ``diff(Heaviside(x),x) = DiracDelta(x)``
-    2) ``integrate(DiracDelta(x-a)*f(x),(x,-oo,oo)) = f(a)`` and
-       ``integrate(DiracDelta(x-a)*f(x),(x,a-e,a+e)) = f(a)``
+    1) ``diff(Heaviside(x), x) = DiracDelta(x)``
+    2) ``integrate(DiracDelta(x - a)*f(x),(x, -oo, oo)) = f(a)`` and
+       ``integrate(DiracDelta(x - a)*f(x),(x, a - e, a + e)) = f(a)``
     3) ``DiracDelta(x) = 0`` for all ``x != 0``
-    4) ``DiracDelta(g(x)) = Sum_i(DiracDelta(x-x_i)/abs(g'(x_i)))``
+    4) ``DiracDelta(g(x)) = Sum_i(DiracDelta(x - x_i)/abs(g'(x_i)))``
        Where ``x_i``-s are the roots of ``g``
+    5) ``DiracDelta(-x) = DiracDelta(x)``
 
     Derivatives of ``k``-th order of DiracDelta have the following property:
 
-    5) ``DiracDelta(x,k) = 0``, for all ``x != 0``
+    6) ``DiracDelta(x, k) = 0``, for all ``x != 0``
+    7) ``DiracDelta(-x, k) = -DiracDelta(x, k)`` for odd ``k``
```

Comment:
@jksuom, have I captured the right behavior here in 7 and 8?

Reply:
Yes, I think so. These are distributions, not functions, but they obey the same rules as functions.